tools/topology/topology2/cavs-rt5682.conf

```diff
@@ -107,10 +108,12 @@ Object.Dai {
 	SSP."1" {
 		id 		$SPK_ID
 		dai_index	1
-		direction	"playback"
+		direction	"duplex"
 		name		SSP1-Codec
 		default_hw_conf_id	0
 		sample_bits		32
+		# used for ref cap
+		quirks			"lbm_mode"
```

Comment:
why are we using LBM mode if there's a demux?

Reply:
Hi, @plbossart , the demux is put on capture pipeline to connect with ACE module in the future.

Reply:
can you share the picture of the new topology, will be easier to see. Thanks.

Reply:
Hi, @plbossart 

Here is the picture with copier.module.8.2 added:
https://github.com/thesofproject/sof/pull/6880#issuecomment-1376031879

Reply:
Are you saying that copier.module.8.2 would have at some point additional sinks connected? I am struggling between what you want now and what will be done later.

Reply:
Hi, @plbossart , yes, that's my current thoughts, something like this (sorry for the ugly flow chart):
```mermaid
graph TD;
   SSP.IN-->copier.module.8.2;
   copier.module.8.2 --> echo-ref-PCM;
   copier.module.8.2 --> AEC;
    subgraph ide1 [future]

   PDM --> AEC;
   AEC --> DMIC48K;
end
```